lib/SearchAndSort/ConnectedSource/StripesConnectedSource.js

```diff
@@ -23,8 +25,19 @@ export default class StripesConnectedSource {
     return res;
   }
 
+  /**
+   * when there are > 10k results to a search, the `totalRecords`
+   * value comes back as `999999999`, so we use that value to indicate
+   * that the count is, in fact, undefined vs returning null to indicate
+   * that the count has not been calculated.
+   */
   totalCount() {
-    const res = this.recordsObj.hasLoaded ? this.recordsObj.other.totalRecords : null;
+    let res;
+    if (!this.recordsObj.hasLoaded) {
+      res = null;
+    } else {
+      res = this.recordsObj.other.totalRecords !== 999999999 ? this.recordsObj.other.totalRecords : undefined;
```

Comment:
maybe to use `UNKNOWN_RECORDS_COUNT` since it's declared above

Reply:
Srsly, what is my problem with this constant?!?

Reply:
I think we all have the same problems from time to time, that's why we need a code review process :rescue_worker_helmet: